front/app/modules/commercial/granular_permissions/admin/components/UserFieldSelection/UserFieldSelection.tsx

```diff
@@ -134,118 +139,156 @@ const UserFieldSelection = ({
     !permission.attributes.global_custom_fields ||
     permission.attributes.permitted_by === 'everyone_confirmed_email';
 
+  const featureDisabled = !permissionsCustomFieldsEnabled;
```

Comment:
Could we use the` !permissionCustomFieldsEnabled ` directly in the Tippy component? I feel like it should be descriptive enough that we don't need another variable :thinking:

Reply:
Yeah, originally I had a whole load of other logic so when I stripped it out, it now makes no sense to do this. Will change it